app/assets/javascripts/base.js.erb

```diff
@@ -6,6 +6,16 @@
 //= require honeybadger-js/dist/honeybadger.js
 //= require ahoy.js/dist/ahoy.js
 //= require serviceworker-companion
+//= require timemachine.js
+
+<% if Rails.env.test? %>
```

Comment:
I'm sure there is a much better place to put this. @vaidehijoshi or @nickytonline please feel free to jump in, I am not very familiar with our js organization.